github-electron/github-electron.d.ts

```diff
@@ -822,6 +868,203 @@ declare namespace Electron {
+		 */
+		fitToView?: boolean;
+		/**
+		 * Offset of the emulated view inside available space (not in fit to view mode)
+		 * Default: {x: 0, y: 0}
+		 */
+		offset?: {
+			/**
+			 * Set the x axis offset from top left corner
+			 */
+			x: number;
+			/**
+			 * Set the y axis offset from top left corner
+			 */
+			y: number;
+		};
+		/**
+		 * Scale of emulated view inside available space (not in fit to view mode)
+		 * Default: 1
+		 */
+		scale: number;
+	}
+
+	interface SendInputEvent {
+		type: 'mouseDown' | 'mouseUp' | 'mouseEnter' | 'mouseLeave' | 'contextMenu' | 'mouseWheel' | 'mouseMove' | 'keyDown' | 'keyUp' | 'char';
+		modifiers: ('shift' | 'control' | 'alt' | 'meta' | 'isKeypad' | 'isAutoRepeat' | 'leftButtonDown' | 'middleButtonDown' | 'rightButtonDown' | 'capsLock' | 'numLock' | 'left' | 'right')[];
+	}
+
+	interface SendInputKeyboardEvent extends SendInputEvent {
+		keyCode: string;
+	}
+
+	interface SendInputMouseEvent extends SendInputEvent {
+		x: number;
+		y: number;
+		button?: 'left' | 'middle' | 'right';
+		globalX?: number;
+		globalY?: number;
+		movementX?: number;
+		movementY?: number;
+		clickCount?: number;
+	}
+
+	interface SendInputMouseWheelEvent extends SendInputEvent {
+		deltaX?: number;
+		deltaY?: number;
+		wheelTicksX?: number;
+		wheelTicksY?: number;
+		accelerationRatioX?: number;
+		accelerationRatioY?: number;
+		hasPreciseScrollingDeltas?: number;
+		canScroll?: boolean;
+	}
+
+	/**
+	 * Debugger API serves as an alternate transport for remote debugging protocol.
+	 */
+	interface Debugger extends NodeJS.EventEmitter {
+		/**
+		 * Attaches the debugger to the webContents.
+		 * @param protocolVersion Requested debugging protocol version.
+		 */
+		attach(protocolVersion?: string): void;
+		/**
+		 * @returns Whether a debugger is attached to the webContents.
+		 */
+		isAttached(): boolean;
+		/**
+		 * Detaches the debugger from the webContents.
+		 */
+		detach(): void;
+		/**
+		 * Send given command to the debugging target.
+		 * @param method Method name, should be one of the methods defined by the remote debugging protocol.
+		 * @param commandParams JSON object with request parameters.
+		 * @param callback Response defined by the ‘returns’ attribute of the command description in the remote debugging protocol.
+		 */
+		sendCommand(method: string, commandParams?: any, callback?: (error: Error, result: any) => void): void;
+		/**
+		 * Emitted when debugging session is terminated. This happens either when
+		 * webContents is closed or devtools is invoked for the attached webContents.
+		 */
+		on(event: 'detach', listener: (event: any, reason: string) => void): this;
+		/**
+		 * Emitted whenever debugging target issues instrumentation event.
+		 * Event parameters defined by the ‘parameters’ attribute in the remote debugging protocol.
+		 */
+		on(event: 'message', listener: (event: any, method: string, params: any) => void): this;
+		on(event: string, listener: Function): this;
```

Comment:
Unfortunatelly, this is a pimitation of TypeScript, the generic case is required :(